recipes/log.c/all/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+cmake_minimum_required(VERSION 3.16)
```

Comment:
Please lower the minimum required version to 3.0 or 3.4 
Depending on using `CMAKE_WIN32_EXPORT_ALL_SYMBOLS`.